### ISSUE TITLE
luTooltipWhenEllipsis and keyboard navigation

### ISSUE THREAD

**Issue by vvalentin-lucca** [LuccaSA/lucca-front#1868]
The tabindex that is added to elements to make them focusable should be removed when tooltips are not needed.

However, this calculation seems to be done on interaction (to avoid listening the resize of the window) and it is therefore already too late when you navigate with the keyboard.

**Comment by Supamiu** [comment]
Do we have any news on that? I remember us talking about ellipsis and tooltips in general but I don't know if it includes this kind of stuff or not.

**Comment by vvalentin-lucca** [comment]
Always relevant, and yes, that's what we're going to try and fix together.

**Comment by jeremie-lucca** [comment]
I guess it was fixed on tooltip/ellipsis rework!
